doc/API/Bills.md

```diff
@@ -132,6 +132,66 @@ Output:
 }
 ```
 
+### `get_bill_graph`
+
+Retrieve a graph associated with a bill
+
+Route: `/api/v0/bills/:id/graphs/:graph_type
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits?from=1517443200
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits?from=1517443200&to=1517788800
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/monthly
+```
+
+Output:
+
+(image)
+
+### `get_bill_graphdata`
+
+Retrieve the data used to draw a graph so it can be rendered in an external system
+
+Route: `/api/v0/bills/:id/graphdata/:graph_type
```

Comment:
Should this be called `/api/v0/bills/:id/data/:type`

Also missing the backtick on the line.

Reply:
I think it's better as graphdata, because the data is pre-processed for use on a graph (number of data points is reduced).  The graph code was actually refactored so that this API call, the graph API calls and the graphs themselves all call the same functions to get data.

Reply:
Ok no worries.